terraform/azure/modules/1_nbs6/agw-private/main.tf

```diff
@@ -0,0 +1,284 @@
+# Re-used variables
+locals {
+  backend_address_pool_name      = "${var.resource_prefix}-agw-backend-pool"
+  frontend_port_name             = "${var.resource_prefix}-agw-frontend-port"
+  frontend_port_name_http        = "${var.resource_prefix}-agw-frontend-http-port"
+  frontend_ip_configuration_name = "${var.resource_prefix}-agw-frontend-ip-configuration"
+  https_setting_name             = "${var.resource_prefix}-agw-backend-https-settings"
+  listener_name                  = "${var.resource_prefix}-agw-https-listener"
+  listener_name_http             = "${var.resource_prefix}-agw-http-listener"
+  probe_name                     = "${var.resource_prefix}-agw-custom-probe"
+  cert_name                      = "${var.resource_prefix}-agw-cert"
+  redirect_configuration_name    = "${var.resource_prefix}-agw-redirect-configuration"
+}
+
+
+
+### Deploy Public App Gateway ###
+
+# Configure Public IP for App Gateway
```

Comment:
The word and name public is in many of the descriptions and resource names. Please ensure these are correct for this file